android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -309,20 +311,14 @@ public void finishNativeInitialization() {
         BraveSyncReflectionUtils.showInformers();
 
         if (BraveConfig.P3A_ENABLED) {
-            // // P3A informer should be shown not for the updated users only
-            // if (!PackageUtils.isFirstInstall(this)) {
-            //     BraveP3AInformers.show();
```

Comment:
PR looks good :+1: 

The only notice is to remove `brave/android/java/org/chromium/chrome/browser/informers/BraveP3AInformers.java`, as it is not used anymore, and it's functions are implemented in `brave/android/java/org/chromium/chrome/browser/onboarding/P3aOnboardingActivity.java`